core/client/src/in_mem.rs

```diff
@@ -362,6 +362,10 @@ impl<Block: BlockT> blockchain::Backend<Block> for Blockchain<Block> {
 	fn leaves(&self) -> error::Result<Vec<Block::Hash>> {
 		Ok(self.storage.read().leaves.hashes())
 	}
+
+	fn children(&self, _parent_hash: Block::Hash) -> error::Result<Vec<Block::Hash>> {
+		unimplemented!()
```

Comment:
don't we need this?

Reply:
No, the `in_mem` backend is only used in some special tests. As long as we don't need the children in these tests, we don't need to implement it.